docs/fundamentals/networking/ipv6-overview.md

```diff
@@ -0,0 +1,208 @@
+
+## Disable or enable IPv6
+
+To use the IPv6 protocol, ensure that you are running a version of the operating system that supports IPv6 and ensure that the operating system and the networking classes are configured properly.
+
+### Configuration Steps
+
+ The following table lists various configurations
+
+| OS IPv6 enabled? | Code IPv6 enabled? | Description                                                                                            |
+|------------------|--------------------|--------------------------------------------------------------------------------------------------------|
+| ❌ No            | ❌ No               | Can parse IPv6 addresses.                                                                              |
+| ❌ No            | ✔️ Yes             | Can parse IPv6 addresses.                                                                              |
+| ✔️ Yes          | ❌ No               | Can parse IPv6 addresses and resolve IPv6 addresses using name resolution methods not marked obsolete. |
+| ✔️ Yes          | ✔️ Yes             | Can parse and resolve IPv6 addresses using all methods including those marked obsolete.                |
+
+For .NET Framework version 1.1 and earlier, the value of the **ipv6 enabled** configuration switch specifies whether members of the <xref:System.Net.Dns?displayProperty=nameWithType> class return IPv6 addresses.
+
+For .NET Framework version 2.0 and later, if Windows supports IPv6, then members of the <xref:System.Net.Dns?displayProperty=nameWithType> class, (for example, the <xref:System.Net.Dns.GetHostEntry%2A?displayProperty=nameWithType> method), will return IPv6 addresses with one limitation. Obsolete members of the DNS <xref:System.Net.Dns?displayProperty=nameWithType> (for example, the <xref:System.Net.Dns.Resolve%2A?displayProperty=nameWithType> method) will read and recognize the value in the configuration file for the ipv6 enabled setting.
+
+To enable the IPv6 support for all classes in the `System.Net` namespace, you must modify the computer configuration file or the configuration file for the application. The configuration file for an application has precedence over the computer configuration file.
+
+To modify a Windows-specific environment configuration file, *machine.config*,  set the switch in a configuration file as follows:
+
+```xml
+<system.net>
+    <settings>
+        <ipv6 enabled="true"/>
+        <!-- Omitted for brevity -->
+    </settings>
+    <!-- Omitted for brevity -->
+</system.net>
+```
+
+> [!NOTE]
+> IPv6 is enabled by default for .NET Framework version 2.0 and later. For .NET Framework version 1.1 and earlier, IPv6 is disabled by default.
```

Comment:
This entire section appears to be .NET Framework specific. What is the corresponding information for .NET Core 1 - 3, and .NET 5 - 7?

Reply:
This is configured with the env var. I'll update this section too.